Tests/LibJS/test-test262.cpp

```diff
@@ -188,7 +192,12 @@ class Test262RunnerHandler {
 
     String read_all()
     {
-        return String(m_input->read_all().bytes(), Chomp);
+        auto all_output_or_error = m_input->read_all();
+        if (all_output_or_error.is_error()) {
+            warnln("Got error: {} while reading runner output", all_output_or_error.error());
+            return ""sv;
+        }
+        return String(all_output_or_error.value().bytes(), Chomp);
```

Comment:
Can we propagate errors here?

Reply:
We could propagate, but again failure is perfectly valid if there is no output or the pipe was already closed.
Then we would just handle the error one layer up.

I guess this program might just be an exception where failure is allowed in a couple of cases.